packages/react-devtools-shared/src/devtools/views/Profiler/HookChangeSummary.js

```diff
@@ -0,0 +1,185 @@
+import * as React from 'react';
```

Comment:
This file is missing Flow typings, you can trigger it by adding a comment header, similar to other files:

```
/**
 * Copyright (c) Meta Platforms, Inc. and affiliates.
 *
 * This source code is licensed under the MIT license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * @flow
 */
```